frontend/src/storage/exampleExpeditionData.json

```
@@ -46,7 +46,7 @@
             "difficulty": "Średnie",
             "options": ["Tak", "Nie"],
             "points": 20,
-            "next": [3, 4],
+            "next": [3, 4, -2],
```

```suggestion
            "next": [-2],
```

Błąd który został tutaj od pierwszej wersji JSONa, można go od razu złapać.

---

Faktycznie, skoro to już koniec grafu to powinno być tylko id końca. Za chwilę poprawię